pyproject.toml

```
@@ -1,7 +1,51 @@
+[project]
+name = "lsst-s3daemon"
+description = "A daemon to maintain a connection pool for bursty S3 data transfers."
+license = {text = "GPLv3+ License"}
```

```suggestion
license = "GPL-3.0-or-later"
```
and add `license-files` line here.